app/src/main/java/org/greenstand/android/TreeTracker/login/LoginFragment.kt

```diff
@@ -0,0 +1,125 @@
+package org.greenstand.android.TreeTracker.login
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import org.greenstand.android.TreeTracker.R
+import org.greenstand.android.TreeTracker.activities.ImageCaptureActivity
+import org.greenstand.android.TreeTracker.utilities.CameraHelper
+import org.greenstand.android.TreeTracker.utilities.mainActivity
+import org.koin.android.viewmodel.ext.android.viewModel
+import timber.log.Timber
+
+class LoginFragment : Fragment() {
+
+//    private lateinit var bindings: FragmentLoginBinding
+
+    private val vm: LoginViewModel by viewModel()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setContent {
+                LoginView(viewModel = vm, navController = findNavController())
+            }
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+
+        mainActivity().bindings.toolbarTitle.apply {
+            setText(R.string.greenstand_welcome_text)
+            setTextColor(ContextCompat.getColor(context, R.color.black))
+        }
```

Comment:
Looks like you took this from the old version. I think we should handle any toolbars (any UI) in compose. There are a few weird practices at the Activity/Fragment level that the legacy code does, so we should feel free to find other solutions.

Reply:
Great to know! Will update this appropriately.